### ISSUE TITLE
NTR, pulmo: traction bronchiectasis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5585]
**Preferred term label:**
traction bronchiectasis

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
CT findings represent irregular bronchial dilatation caused by surrounding retractile pulmonary fibrosis. Dilated airways are usually identifiable as such but may be seen as cysts. 

Comment: The juxtaposition of numerous cystic airways may make the distinction from “pure” fibrotic honeycombing difficult. Also the differentation to peribronchovascular interstitial thickening can be difficult, but they can be distinguished on the basis of symptoms or pulmonary function abnormalities. In addition, patients with traction bronchiectasis have abnormal thick-walled and dilated bronchi that often appear much larger than the adjacent pulmonary artery braches, with a bronchoarterial ratio exceeding 1. This results in the appearance of an additional finding, the signet ring sign (HP:0032175)

PMID: 18195376, PMID: 3786716

**Parent term (use hpo.jax.org/app)**
Abnormal bronchus morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sarcoidosis, ORPHA:797
Idiopathic pulmonary fibrosis (ORPHA:2032, OMIM 178500)
Hypersensitivity pneumonitis (ORPHA:31740, OMIM 145300)

**Your nano-attribution (ORCID)**
LMU: crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
I modified the definition to provide more background:

Distortion of the bronchial airways due to mechanical traction on the bronchi resulting from fibrosis of the surrounding lung parenchyma. CT findings represent irregular bronchial dilatation caused by surrounding retractile pulmonary fibrosis. Dilated airways are usually identifiable as such but may be seen as cysts.


If this needs further work please reopen @mgriese, @katrinknoflach
